kitsune/tidings/utils.py

```diff
@@ -94,3 +99,15 @@ def import_from_setting(setting_name, fallback):
 
 # Here to be imported by others:
 reverse = import_from_setting("TIDINGS_REVERSE", django_reverse)  # no QA
+
+
+def get_class(module_name, class_name):
+    """
+    Convenience function for extracting a class from the given module name using
+    the given class name. Raises a DisallowedModule exception if the module name
+    is outside of kitsune.
+    """
+    if not module_name.startswith("kitsune."):
```

Comment:
This is really important for security. I wonder if we need to lock this down even more? I don't think so, but would like to hear your thoughts. For example, I was thinking that we could lock it down even more by checking that the `module_name` is one of the installed apps, or something like that.